src/DataAccess/Migrations/Version20240528083322.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\AddressChangeContext\DataAccess\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240528083322 extends AbstractMigration {
+
+	public function getDescription(): string {
+		return 'Add indexes for full text search on the Address table';
+	}
+
+	public function up( Schema $schema ): void {
+		$table = $schema->getTable( 'address' );
+		$table->addIndex( [ 'first_name' ], 'idx_first_name_address_change' );
+		$table->addIndex( [ 'last_name' ], 'idx_last_name_address_change' );
+		$table->addIndex( [ 'street' ], 'idx_street_address_change' );
+		$table->addIndex( [ 'postcode' ], 'idx_postcode_address_change' );
+		$table->addIndex( [ 'city' ], 'idx_city_address_change' );
```

Comment:
To avoid naming clashes between indexes I usually mix in some letter after the `idx_`, eg. `idx_m_` for memberships. This could be `idx_ac_`